recipe/emailpassword/recipeImplementation.go

```diff
@@ -158,14 +158,28 @@ func MakeRecipeImplementation(querier supertokens.Querier) epmodels.RecipeInterf
 		}
 	}
 
-	updateEmailOrPassword := func(userId string, email, password *string, userContext supertokens.UserContext) (epmodels.UpdateEmailOrPasswordResponse, error) {
+	updateEmailOrPassword := func(userId string, email, password *string, applyPasswordPolicy *bool, userContext supertokens.UserContext) (epmodels.UpdateEmailOrPasswordResponse, error) {
 		requestBody := map[string]interface{}{
 			"userId": userId,
 		}
 		if email != nil {
 			requestBody["email"] = email
 		}
 		if password != nil {
+			if applyPasswordPolicy == nil || *applyPasswordPolicy {
+				formFields := getEmailPasswordConfig().SignUpFeature.FormFields
+				for i := range formFields {
```

Comment:
```suggestion
				for i := range formFields {
					if formFields[i].ID == "password" {
						err := formFields[i].Validate(*password)
						if err != nil {
							errResponse := epmodels.PasswordPolicyViolatedError{
								FailureReason: err,
							}
							return epmodels.UpdateEmailOrPasswordResponse{PasswordPolicyViolatedError: &errResponse}, nil
						}
					}
					break;
				}
```

Should I add a break statement here, there can be a lot of fields but as soon as password field is validated we can break